### PR TITLE
Add initial line number usage

### PR DESCRIPTION
**Pull request by char-gu**

### CODE REVIEW

File: tiller_python_service/controller.py [hunk 1]
```diff
@@ -18,33 +22,41 @@ def should_generate_fix(payload):
...
-    pull_request_message = open_pull_request(ssh_url, new_branch_name, source_branch_name)
+    pull_request_message = open_pull_request(
+        ssh_url, new_branch_name, source_branch_name
+    )
 
     notify_pr_commenter_of_proposal(pr_number, comment_id, pull_request_message)
 
+
 def add_these_numbers(num1, num2):
     return num1 + num2
```
Comment (char-gu) [inline tiller_python_service/controller.py:57]: remove this function

Comment (char-gu) [reply tiller_python_service/controller.py:57]: remove this function help tiller

File: tiller_python_service/controller.py [hunk 2]
```diff
@@ -26,33 +31,41 @@ def should_generate_fix(payload):
...
 
-    pull_request_message = open_pull_request(ssh_url, new_branch_name, source_branch_name)
+    pull_request_message = open_pull_request(
+        ssh_url, new_branch_name, source_branch_name
+    )
 
     notify_pr_commenter_of_proposal(pr_number, comment_id, pull_request_message)
 
+
 def add_these_numbers(num1, num2):
```
Comment (char-gu) [inline tiller_python_service/controller.py:65]: remove this function

help tiller

File: tiller_python_service/controller.py [hunk 3]
```diff
@@ -1,6 +1,9 @@
 # from test_blob import blob
-from git_actions import clone_and_create_new_branch,git_add_commit_push,open_pull_request
-from git_actions import clone_and_create_new_branch, git_add_commit_push
+from git_actions import (
+    clone_and_create_new_branch,
```
Comment (AnotherOctopus) [inline tiller_python_service/controller.py:3]: rename this help tiller

File: tiller_python_service/controller.py [hunk 4]
```diff
@@ -9,12 +12,14 @@
...
-       ast.parse(code)
-   except SyntaxError:
-       return False
-   return True
+    try:
+        ast.parse(code)
+    except SyntaxError:
+        return False
+    return True
+
```
Comment (AnotherOctopus) [inline tiller_python_service/controller.py:22]: delete the whitespace here

Comment (AnotherOctopus) [reply tiller_python_service/controller.py:22]: delete the whitespace here help tiller

**Pull request merged**